packages/axios-extension/src/abap/adt-catalog/services/businessobjects-service.ts

```diff
@@ -38,7 +38,7 @@ export class BusinessObjectsService extends AdtService {
                 query: `*`,
                 maxResults: maxResults || 10000,
                 objectType: 'BDEF',
-                releaseStatus: 'USE_IN_CLOUD_DEVELOPMENT'
+                releaseState: 'USE_IN_CLOUD_DEVELOPMENT'
```

Comment:
Not related to this line. But the method signature. 

Default value is preferred over optional parameter. E.g. you can write

`getBusinessObjects(maxResults = 10000)`

instead of 

`getBusinessObjects(maxResults?: number)`

Hence, you also don't need

`maxResults: maxResults || 10000,`

You can use:

```
 params: {
                operation: 'quickSearch',
                query: `*`,
                maxResult,
                ...
}
```

Reply:
Thanks, I'll make the change